nym-node/nym-node-http-api/src/router/api/v1/gateway/client_interfaces/wireguard/mod.rs

```diff
@@ -214,11 +219,11 @@ mod test {
         assert_eq!(response.status(), StatusCode::OK);
         assert!(!registration_in_progress.is_empty());
 
-        let ClientRegistrationResponse::PendingRegistration {
+        let ClientRegistrationResponse::PendingRegistration(RegistrationData {
             nonce,
             gateway_data,
             wg_port: 8080,
-        } = serde_json::from_slice(&to_bytes(response.into_body(), usize::MAX).await.unwrap())
+        }) = serde_json::from_slice(&to_bytes(response.into_body(), usize::MAX).await.unwrap())
```

Comment:
Do these two unwraps make it possible to crash the gateway with a malformed request?

Reply:
This is inside a unit test so no problem of crash here